modules/likes.php

```diff
@@ -968,6 +968,10 @@ function get_options() {
 	 * similar logic and filters apply here, too.
 	 */
 	function is_likes_visible() {
+		if ( Jetpack_Sync_Settings::is_syncing() ) {
```

Comment:
I think this could be moved lower to the line where we have 
`if ( is_feed() || is_preview() || $comment_popup )` so it reads
`if (  Jetpack_Sync_Settings::is_syncing() || is_feed() ...`